modules/web/src/app/dynamic/enterprise/quotas/default-project-quota/template.html

```diff
@@ -0,0 +1,51 @@
+<!--
+Copyright 2023 The Kubermatic Kubernetes Platform contributors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<div fxLayout="row"
+     fxLayoutAlign="space-between">
+  <div fxFlex="16%"
+       fxLayoutAlign=" center"
+       class="entry-label">
+    <span>Default Project Quota</span>
+    <i class="km-icon-info km-pointer"
+       matTooltip="Applies to every project. Specific Project Quotas override the Default Project Quota."></i>
+    <km-spinner-with-confirmation class="spinner-icon"
+                                  [isSaved]="isEqual(settings.defaultQuota.quota, apiSettings.defaultQuota?.quota)"></km-spinner-with-confirmation>
+  </div>
+
+  <div fxLayout="row wrap"
+       fxLayoutAlign="flex-end"
+       fxLayoutGap="16px">
+    <km-number-stepper [(ngModel)]="settings.defaultQuota.quota.cpu"
+                       (ngModelChange)="onSettingsChange()"
+                       label="CPU"
+                       min="0">
```

Comment:
the min = 0 ?? what would happen if the value 0 also we need to check if the value less than 0 else than to show an error we need to prevent the update request
![Screenshot from 2023-01-19 08-15-48](https://user-images.githubusercontent.com/85109141/213361536-92105a8d-0eca-4717-a7e3-2f88b22c0adb.png)